pixi/src/checks/PageExperienceCheck.js

```diff
@@ -35,21 +40,33 @@ export default class PageExperienceCheck {
     }
   }
 
+  createFieldData(fieldData, id) {
+    const metric = fieldData[id];
+    const score = metric.percentile / METRICS_MAX[id];
```

Comment:
Looks like this logic is not correct.
The better the page the smaller the value, but that does not match the logic here